pdr_backend/analytics/test/test_get_predictions_info.py

```diff
@@ -0,0 +1,176 @@
+from unittest.mock import patch
+from enforce_typing import enforce_types
+import polars as pl
+from pdr_backend.lake.table_pdr_predictions import (
+    _object_list_to_df,
+    predictions_schema,
+)
+from pdr_backend.util.timeutil import timestr_to_ut
+from pdr_backend.analytics.get_predictions_info import get_predictions_info_main
+from pdr_backend.ppss.ppss import mock_ppss
+
+
+@enforce_types
+@patch(
+    "pdr_backend.analytics.get_predictions_info.get_feed_summary_stats",
+    spec=pl.DataFrame,
+)
+@patch("pdr_backend.analytics.get_predictions_info.GQLDataFactory.get_gql_dfs")
+def test_get_predictions_info_main_mainnet(
+    mock_getPolars,
```

Comment:
@KatunaNorbert, all functions and parameters need to be named using snake case.
Please be mindful to separate words.

```
mock_getPolars => mock_get_polars
mock_getstats => mock_get_stats
```